app/services/orangelight_reindexer.rb

```diff
@@ -0,0 +1,48 @@
+# frozen_string_literal: true
+class OrangelightReindexer
+  # Reindexes resources indexed into Orangelight by sending record updated messages
+  # for all complete resources.
+  # @param optional logger [Logger] instance of logger class
+  def self.reindex_orangelight(logger: Logger.new(STDOUT))
+    new(logger: logger).reindex_orangelight
+  end
+
+  attr_reader :logger, :ogm_repo_path
+  def initialize(logger:)
+    @logger = logger
+  end
+
+  def reindex_orangelight
+    all_orangelight_resources.each do |resources|
+      resources.each do |resource|
```

Comment:
Why are there two arrays?

Reply:
it was copied from the geo service. I un-nested them.